ui-admin/src/study/participants/enrolleeView/EnrolleeProfile.tsx

```diff
@@ -286,7 +323,7 @@ export function FormRow(
   }
 ) {
   return <>
-    <div className="w-25 fw-bold mb-4 mt-2">
+    <div className="w-25 fw-bold mb-4 mt-2" aria-label={title}>
       {title}
     </div>
     <div className="w-75 mb-4">
```

Comment:
Nice work on the aria-labels up above.

I don't think adding the aria label here addresses the accessibility concern though.  The accessibility concern is that if you're just using divs, there's nothing that tells a blind reader that the title is connected to the content.  So what you want to do is use labels and htmlFor so the relation between the two elements is clear.  So you can use  `const contentId =  useId()` and then:

```suggestion
    <label className="w-25 fw-bold mb-4 mt-2" htmlFor={contentId}>
      {title}
    </div>
    <div className="w-75 mb-4" id={contentId}>
```

Reply:
Ahhh, I understand. I have not done accessibility stuff before so this is new to me. Thanks

Reply:
Does the aria id need to be on the `input` as well? Or is your code enough to make it accessible?

Reply:
yes, that's right, the id should be on the input -- good catch!

Reply:
Hmm, then there's an issue: There are multiple inputs to one label, e.g., `Name` has `Given Name` and `Family Name`. As far as I can tell, without doing some field weirdness (`<fieldset>`), `aria-label` seems like the best we can do.